src/config/config.c

```diff
@@ -150,7 +149,7 @@ int Config_SetIntraOperationParallelism(RedisModuleString *num_threads_string) {
 int Config_SetModelChunkSize(RedisModuleString *chunk_size_string) {
     long long val;
     int result = RedisModule_StringToLongLong(chunk_size_string, &val);
-    if (result != REDISMODULE_OK || val <= 0) {
+    if (result != REDISMODULE_OK || val <= 0 || val > REDISAI_DEFAULT_MODEL_CHUNK_SIZE) {
```

Comment:
why the addition of this condition?

Reply:
We need to limit the `chunk_size`, since Redis cannot reply with blobs that are larger than 512MB, right?

Reply:
https://github.com/redis/redis-doc/pull/1653#issuecomment-922891303
see here. You can configure the server to support bigger values